CHANGELOG.next.toml

```diff
@@ -16,3 +16,47 @@ message = "Add codegen version to generated package metadata"
 references = ["smithy-rs#1612"]
 meta = { "breaking" = false, "tada" = false, "bug" = false }
 author = "unexge"
+
+[[smithy-rs]]
+message = """
+Support granular control of specifying runtime crate versions.
+
+For code generation, the field `runtimeConfig.version` in smithy-build.json has been removed.
+The new field `runtimeConfig.verisions` is an object that keys are runtime crate names (e.g. aws-smithy-http),
+and values are their versions user specifying.
+
+If you previous set `version = "DEFAULT"`, then you're lucky.
+The new behaviour automatically detects and injects the version for you by checking the version of the code generator.
+You don't need to add any new configuration.
```

Comment:
Is this true? I don't recognize it being implemented.

Reply:
Hmm… I can't remember what I was writing about. Anyway, fixed with b31e0b4f